### PR TITLE
Search: search results show a blank instead of the page title

### PR DESCRIPTION
**Pull request by Bazif-Khan**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

fixes: https://github.com/appsody/website/issues/474

### CODE REVIEW

File: src/components/wideBlogPost.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
-import React, { Component } from "react";
+import React from "react";
 import { Link } from "gatsby";
 
-const WideBlogPost = ({title, subtext, author, date, image, slug}) => (
```
Comment (ddunn2) [inline src/components/wideBlogPost.js]: Why did you remove this? 

I'm assuming it's because we're not using it? Just wanted to confirm :)

Comment (Bazif-Khan) [reply src/components/wideBlogPost.js]: Yeah, it's not being used.
